data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/LambdaProcessorConfig.java

```diff
@@ -21,6 +21,17 @@
         "It supports both synchronous and asynchronous invocations based on your use case.")
 public class LambdaProcessorConfig extends LambdaCommonConfig {
   static final String DEFAULT_INVOCATION_TYPE = "request-response";
+  protected static final int DEFAULT_CIRCUIT_BREAKER_RETRIES = 0;
+  protected static final long DEFAULT_CIRCUIT_BREAKER_WAIT_INTERVAL_MS = 1000;
+
+  @JsonProperty("circuit_breaker_retries")
```

Comment:
Circuit breaker should be something that we control and configure within `data-prepper-config.yaml` rather than manage within the pipeline YAML itself.